mods/tnt/license.txt

```diff
@@ -64,3 +64,45 @@ rights may limit how you use the material.
 
 For more details:
 http://creativecommons.org/licenses/by-sa/3.0/
+
+
+Other licensing used for media
```

Comment:
No need for this as this text also comes under 'Licenses of media'.

Reply:
> No need for this as this text also comes under 'Licenses of media'.

line removed